### DISCUSSION TITLE
[FR] Allowing `Polygon` and `Vector{Polygon}` as marker type for `scatter`

### DISCUSSION THREAD

**Discussion by Datseris** [JuliaPlots/Makie.jl#4195] [Ideas]
Hello,

it would be super useful if this is possible. The `meshscatter` approach does not work for many applications, e.g. scattering different polygons at different positions.

Can you please outline what is necessary for this to be possible?

**Comment by ffreyer** [comment]
Oh this has actually been implemented. Closing as completed
